cranelift/codegen/src/isa/s390x/inst/emit.rs

```diff
@@ -21,6 +21,7 @@ pub fn mem_finalize(
     have_d12: bool,
     have_d20: bool,
     have_pcrel: bool,
+    have_unaligned_pcrel: bool,
```

Comment:
This list of flags is long enough now that it's starting to feel a bit error-prone; and e.g. the diff adding `false` in the right places is hard to review carefully. Could we turn this into a struct of flags?

Reply:
Agreed, that makes sense.  Like this?